### ISSUE TITLE
Is this a problem with the data set

### ISSUE THREAD

**Issue by vaeXu** [anatolix/keras_Realtime_Multi-Person_Pose_Estimation#32]
I used **two days** of training ./rmpe_dataset_server ./train_dataset.h5 5555,but it keeps:
Total samples: 2645
Epoch 1
curr_sample/total_samples/curr_epoch = 1/2645/1
What is the reason for this?Is the amount of data too large? What I use is the provided COCO2014 data set.If someone knows the reason, please answer your questions. Be grateful.

**Comment by anatolix** [comment]
That is not correct dataset size. Something is wrong. Looks more like validation dataset

**Comment by abinjoabraham** [comment]
@anatolix @GAVANXU Is it mandatory to go with the coco2014 data?? I was using the latest version of the dataset coco2017

**Comment by anatolix** [comment]
No it is not, I also train with 2017. But original work uses coco2014 for some reason. Imho 2014 and 2017 datasets are same for 90% pictures.
